### PR TITLE
First bug  #351

### PR DESCRIPTION
**Pull request by ciobancai**

### CODE REVIEW

File: relengapi/docs/deployment/slaveloan.rst [hunk 1]
```diff
@@ -6,11 +6,9 @@ Bugzilla
 
 Slaveloan makes extensive use of Bugzilla's REST API for classifying new loans, reopening slave tracking bugs, etc.
 
-In order to use it, you *must* specify where and what user to use (in the settings file). e.g.::
+::
 
     BUGZILLA_URL = "https://bugzilla-dev.allizom.org/rest/"
-    BUGZILLA_USER = "my.user@example.com"
-    BUGZILLA_PASS = "my.passw0rd"
```
Comment (djmitche) [inline relengapi/docs/deployment/slaveloan.rst]: This doesn't seem to say anything about the API key, and also removes docs about BUGZILLA_USER and BUGZILLA_PASS (which, I think, will still work, right?)

**Comment by djmitche** [comment]
It'd be best to rebase and remove 87f4be3d62f7c84aaf7a96e1935c1094dd63460f and 23cca6b since they are just test commits.

**Comment by Callek** [comment]
At this point I'd also rebase to remove the docs + revert-of-docs ;-)

That said, we do need to update the docs, but they should reflect the code changes you do.

The function that needs changing is https://github.com/mozilla/build-relengapi/blob/master/relengapi/blueprints/slaveloan/bugzilla.py#L21 which explicitly reads the set config values and configures bzrest to use those values. Without that change this does nothing.

File: setup.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
         "Celery>=3.1.16",  # see https://github.com/mozilla/build-relengapi/issues/145
         "alembic>=0.7.0",
         "argparse",
-        "requests",
+        "requests>=1.2.3",
```
Comment (Callek) [inline setup.py]: I am similarly unsure why you're bumping requests version here.

File: relengapi/blueprints/slaveloan/bugzilla.py [hunk 3]
```diff
@@ -17,17 +17,13 @@
 MAX_ALIAS = 15
 DEFAULT_BUGZILLA_URL = "https://bugzilla-dev.allizom.org/rest/"
 
-
```
Comment (Callek) [inline relengapi/blueprints/slaveloan/bugzilla.py]: relengapi/blueprints/slaveloan/bugzilla.py:20:1: E302 expected 2 blank lines, found 1

**Comment by Callek** [comment]
Also needs a doc update, to describe the new config var.

File: relengapi/docs/deployment/slaveloan.rst [hunk 4]
```diff
@@ -9,8 +9,7 @@ Slaveloan makes extensive use of Bugzilla's REST API for classifying new loans,
 In order to use it, you *must* specify where and what user to use (in the settings file). e.g.::
```
Comment (Callek) [inline relengapi/docs/deployment/slaveloan.rst]: s/and what user to use/and an apikey for a user/

File: setup.py [hunk 5]
```diff
@@ -37,8 +37,7 @@
         "IPy",
         "furl",
         "redo",
-        # Temporary freeze until https://github.com/bhearsum/bzrest/pull/3 is fixed
-        "bzrest==0.9",
+        "bzrest==1.1",
```
Comment (Callek) [inline setup.py]: I'd actually like >=1.1 here, incase there are other updates that are useful.

**Comment by Callek** [comment]
:+1: \o/ Thanks for the patch

**Pull request merged**